gio/src/auto/dbus_interface_info.rs

```diff
@@ -19,16 +19,10 @@ glib::wrapper! {
 }
 
 impl DBusInterfaceInfo {
-    #[doc(alias = "g_dbus_interface_info_generate_xml")]
-    pub fn generate_xml(&self, indent: u32, string_builder: &mut glib::String) {
-        unsafe {
-            ffi::g_dbus_interface_info_generate_xml(
-                self.to_glib_none().0,
-                indent,
-                string_builder.to_glib_none_mut().0,
-            );
-        }
-    }
+    //#[doc(alias = "g_dbus_interface_info_generate_xml")]
```

Comment:
would be nice to generate those manually and make them return a String :)

Reply:
If someone needs them we can add them back. I don't understand why anybody is using the `GDbus` bindings as they are (without some high-level API that makes it convenient to use) anyway, but this is even more fringe API than the other parts.

Reply:
Or said otherwise: I'm not going to spend more time on `GDbus` and I still think it was a mistake to add at all without something more useful around it. It's enough of a burden to keep it compile with all the other changes going on :)

Reply:
FWIW, outside GLib these two functions are used by no code inside Debian. The only other references are from autogenerated bindings for various languages, where there's no code in Debian making use of that bindings API either.